packages/cli/mock/src/runMockServer.ts

```diff
@@ -0,0 +1,208 @@
+import { assertNever } from "@fern-api/core-utils";
+import { generateIntermediateRepresentation } from "@fern-api/ir-generator";
+import { TaskContext } from "@fern-api/task-context";
+import { FernWorkspace } from "@fern-api/workspace-loader";
+import {
+    ExampleEndpointCall,
+    ExamplePathParameter,
+    ExampleTypeReference,
+    HttpEndpoint,
+    NameAndWireValue
+} from "@fern-fern/ir-sdk/api";
+import express, { Request, Response } from "express";
+import { ParamsDictionary, Query } from "express-serve-static-core";
+import getPort from "get-port";
+import { IncomingHttpHeaders } from "http";
+import { isEqual } from "lodash-es";
+import qs from "qs";
+import urlJoin from "url-join";
+
+type RequestHandler = (req: Request, res: Response) => void;
+
+export async function runMockServer({
+    context,
+    workspace,
+    port
+}: {
+    context: TaskContext;
+    workspace: FernWorkspace;
```

Comment:
my guess is the entry point here should be IR, not a fern workspace -- that way anything can call into this as long as they have IR